apps/modernization-ui/src/apps/search/patient/add/layout/add-layout.module.scss

```diff
@@ -0,0 +1,23 @@
+@use 'styles/colors';
+@use 'styles/borders';
+
+.addLayout {
+    display: flex;
+    height: calc(100vh - 75px);
+    background-color: colors.$base-white;
+
+    .content {
+        display: flex;
+        width: 100%;
+        flex-direction: column;
+
+        main {
+            display: flex;
+            height: 100%;
+            padding: 1rem 1rem 0 1rem;
+            background-color: colors.$background;
+            overflow: auto;
+            scroll-behavior: smooth;
```

Comment:
suggestion:  The `scroll-behavior` should be consistent across the entire application.  This really belongs in the global stylesheet on the `html` or `body` element.